setup.py

```diff
@@ -57,8 +57,8 @@ def version2pep440(version):
 setup(
       name = "idvc",
       description = 'CCPi DVC Configurator',
-	version = dversion,
-	packages = {'idvc'},
+	  version = dversion,
+	  packages = {'idvc'},
       package_dir = {'idvc': os.path.join('src','idvc')},
```

Comment:
Unfortunately I have been unable to test this locally (I have tried on windows, and on linux [DAaaS]) and I get the same error:

> ModuleNotFoundError: No module named 'idvc.ui'

I even tried adding this here, but it did not help:

```
      package_dir = {'idvc': os.path.join('src','idvc'), 'idvc.ui': os.path.join('src','idvc', 'ui')}
```

I think we will need to look at this together some time and see what is going wrong.

Reply:
yeah, that solved the problem